File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantBatchesType.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantBatchesType {
    @JsonProperty("CanBeCloned")
    public Boolean canBeCloned;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("LastModified")
    public String lastModified;
    @JsonProperty("Name")
    public String name;
}

File: sdks/thunkmetrc-go/wrapper/models/plant_batches_type.go
package models

type PlantBatchesType struct {
    CanBeCloned bool `json:"CanBeCloned,omitempty"`
    Id int64 `json:"Id,omitempty"`
    LastModified string `json:"LastModified,omitempty"`
    Name string `json:"Name,omitempty"`
}
